README.md

```diff
@@ -40,11 +40,3 @@ Please check [platform support](https://github.com/NLog/NLog/wiki/platform-suppo
 Known issues
 ---
 - auto load of NLog extensions won't work yet. Use `<extensions>` (see [docs](https://github.com/NLog/NLog/wiki/Configuration-file#extensions))
-
```

Comment:
> Maybe it's better to rename asp.net core to .net core?

There are still .NET Core examples...

Reply:
Well it is hard to request the guy that wants to avoid using Microsoft Extension Logging, that he should write about how to use it :). I just saw that the chapter didn't make any sense for the current repository.